adapters/nativo/nativo_test.go

```diff
@@ -0,0 +1,21 @@
+package nativo
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+func TestBidderNativo(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderNativo, config.Adapter{
+		Endpoint: "https://exchange.postrelease.com/esi?ntv_epid=7"},
```

Comment:
I suggest using a fake URL here and in all of your JSON test files for maintenance purposes.